android-json-form-wizard/src/main/java/com/vijay/jsonwizard/constants/JsonFormConstants.java

```diff
@@ -143,6 +143,7 @@
     public static final String MIN_DATE = "min_date";
     public static final String MAX_DATE = "max_date";
     public static final String EXPANDED = "expanded";
+    public static final String EXPAND_ON_TEXT_CHANGE = "expandOnTextChange";
```

Comment:
Could we use underscores instead of camelCase for the field? That is more consistent with other fields in native form widgets.